FreeRTOS/iot-reference-esp32c3
# Merge switch to new ota to main

Description
-----------
* Update to LTS 3.0 library. Currently, a fork is used instead of esp-aws-iot. We will update the submodule link after PR is merged to esp-aws-iot.
* Merge SwitchToNewOTA branch to main branch
* Fix xEventGroupWaitBits wait bits in coreMQTT agent manager for device advisor test
* Fix second network context CA cert length for MQTT test

Test Steps
-----------
Run device advisor test and MQTT test should have no problem.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
